### PR TITLE
Allow to define auth token for quay.io

### PR DESCRIPTION
**Pull request by Michkov**
Quay can contain private repositories. Allow to access them using token
in environment variable.

### CODE REVIEW

File: repotracker.ini [hunk 1]
```diff
@@ -9,9 +9,12 @@ topic_prefix = VirtualTopic.eng.repotracker
...
 
 [datagrepper]
 type = container
 repo = quay.io/factory2/datagrepper
-tags = latest stage prod
+
+[secretrepo]
+type = container
+repo = quay.io/factory2/secret
+token = FACTORY2_QUAY_TOKEN
```
Comment (mikebonnet) [inline repotracker.ini:20]: Should this be `token_env`? That's the key referenced in `container.py`.

Comment (Michkov) [reply repotracker.ini]: yes, updated, thanks, also fixed tests and added new ones.

Comment (mikebonnet) [reply repotracker.ini]: Great!

**Comment by mikebonnet** [comment]
Can you fix the way `flake8` is called in `.travis.yml` so the tests succeed?

**Comment by mikebonnet** [comment]
Looks great, thanks!

**Pull request merged**